src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -127,18 +130,18 @@ RBPullUpMethodRefactoring >> checkInstVars [
 { #category : 'preconditions' }
 RBPullUpMethodRefactoring >> checkInstVarsFor: aSelector [
 	"Checks if `aSelector` refers to one of the instance variables and offers to perform
-	`PushUpInstanceVariableRefactoring` refactoring such variables exist."
+	`PullUpInstanceVariableRefactoring` refactoring such variables exist."
 
 	class instanceVariableNames do:
 			[:each |
 			((class whichSelectorsReferToInstanceVariable: each) includes: aSelector) ifTrue:
-					[ (self confirm: ('<1p> refers to #<2s> which is defined in <3p>. Do you want push up variable #<2s> also?' expandMacrosWith: aSelector
+					[ (self confirm: ('<1p> refers to #<2s> which is defined in <3p>. Do you want pull up variable #<2s> also?' expandMacrosWith: aSelector
 								with: each
 								with: class))
-						ifTrue: [ self pushUpVariable: each ]
-						ifFalse: [ self refactoringWarning: 'You are about to push your method without the instance variable it uses.
+						ifTrue: [ self pullUpVariable: each ]
+						ifFalse: [ self refactoringWarning: 'You are about to pull your method without the instance variable it uses.
 						It will bring the system is an inconsistent state. But this may be what you want.
-						So do you want to push up anyway?' ] ]]
+						So do you want to pull up anyway?' ] ]]
```

Comment:
@Ducasse I think you were refering to these warnings that need to be moved to driver.
This PR is getting very long, I would also suggest to merge this once non-negotiables are fixed and then in a next PR fix this and options.

Reply:
I have created [issue#17158](https://github.com/pharo-project/pharo/issues/17158) to handle this